doc/source/setup-helm.rst

```diff
@@ -77,11 +72,10 @@ It should provide output like:
 
    .. code-block:: bash
 
-      Client: &version.Version{SemVer:"v2.6.2", GitCommit:"46d9ea82e2c925186e1fc620a8320ce1314cbb02", GitTreeState:"clean"}
-      Server: &version.Version{SemVer:"v2.6.2", GitCommit:"46d9ea82e2c925186e1fc620a8320ce1314cbb02", GitTreeState:"clean"}
+      Client: &version.Version{SemVer:"v2.4.1", GitCommit:"46d9ea82e2c925186e1fc620a8320ce1314cbb02", GitTreeState:"clean"}
+      Server: &version.Version{SemVer:"v2.4.1", GitCommit:"46d9ea82e2c925186e1fc620a8320ce1314cbb02", GitTreeState:"clean"}
 
-Make sure you have at least version 2.6.2, but not newer (until
-`this bug <https://github.com/kubernetes/helm/issues/3275>`_ is fixed)
+Make sure you have at least version 2.4.1!
```

Comment:
This is now saying that 2.4.1 is okay, but I think 2.7 is the minimum version now?